### ISSUE TITLE
Add to homebrew

### ISSUE THREAD

**Issue by derekperkins** [zab/cli#3]
Make zab installable via homebrew

**Comment by jsonmaur** [comment]
Interesting idea. What would be the benefit? An easy install for users who don't have Node on their system?

**Comment by derekperkins** [comment]
I use brew for 95% of the tools I use, especially command line tools. It's a single place to install + upgrade. There's also the marketing aspect of discoverability.

**Comment by jsonmaur** [comment]
Yeah better discoverability could be a plus. I've never actually heard of a Node module being published to homebrew though. I'll keep this open and look into it later down the road. For now you can just run `brew install node && npm install -g zab`. Not as nice as `brew install zab`, but close :)
